app/src/main/java/com/example/workoutrival/SplashScreen.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.workoutrival
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import android.view.Window
+import android.view.WindowManager
+
+class SplashScreen : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        window.requestFeature(Window.FEATURE_NO_TITLE)
+        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)
+        setContentView(R.layout.activity_splash_screen)
+
+        Handler().postDelayed({
+            startActivity(Intent(this@SplashScreen, MainActivity::class.java))
```

Comment:
Please check service availability...
You may use this endpoint: **http://localhost/actuator/health**  which returns for example: 
```
{
...
     "status" : "UP"
...
}
```